core/src/main/java/org/mule/runtime/core/api/processor/factory/ProcessingStrategyFactory.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.core.api.processor.factory;
```

Comment:
-> `org.mule.runtime.core.api.processor.strategy` (move ProcessingStrategy too in this case) or `org.mule.runtime.core.api.processor`.